pkg/provisioner/ironic/testserver/redfishvirtualmedia.go

```diff
@@ -0,0 +1,33 @@
+package testserver
+
+import (
+	"net/http"
+	"testing"
+)
+
+// RedfishVirtualMediaMock is a test server that implements Redfish Virtual Media semantics
+type RedfishVirtualMediaMock struct {
+	*MockServer
+}
+
+// NewRedfishVirtualMedia builds a redfish virtual media mock server
+func NewRedfishVirtualMedia(t *testing.T) *RedfishVirtualMediaMock {
+	return &RedfishVirtualMediaMock{
+		MockServer: NewTLS(t, "redfish-virtual-media"),
+	}
+}
+
+// Endpoint returns the URL to the server
+func (m *RedfishVirtualMediaMock) Endpoint() string {
+	if m == nil {
```

Comment:
Not sure this is strictly required

Reply:
I've changed to 
```
// Endpoint returns the URL to the server
func (m *RedfishVirtualMediaMock) Endpoint() string {
	return "https://bmcaddress"
}
```

Reply:
I meant to keep just `return m.MockServer.RawEndpoint()`